project/client/like-home/src/routes.js

```diff
@@ -0,0 +1,19 @@
+// import React from 'react'
```

Comment:
is this supposed to be commented out still?

Reply:
For now yes